cli/src/main/java/com/devonfw/tools/ide/tool/graalvm/GraalVm.java

```diff
@@ -29,12 +28,6 @@ public Path getToolPath() {
     return this.context.getSoftwareExtraPath().resolve(getName());
   }
 
-  @Override
-  public VersionIdentifier getInstalledVersion() {
-
-    return super.getInstalledVersion(getToolPath());
```

Comment:
this did not make so much sense:
1. if you override `getInstalledVersion()` you may delegate to `super.getInstalledVersion()` but if you do not override `getInstalledVersion(Path)` the `super.` prefix is only causing confusion.
2. This method was only overridden because of a bug in the original implementation that was overridden here. Since I fixed this bug, I could remove this now.